src/Wordpress/CustomPostType/Map.php

```diff
@@ -246,6 +254,7 @@ function ( $item ) {
 					'terms'      => $item_terms,
 					'link'       => add_query_arg( 'cb-location', $post->ID, get_permalink( $item->ID ) ),
 					'thumbnail'  => $thumbnail ?: null,
```

Comment:
Hier könnte man doch auch $images['thumbnail'][0] verweden. Bei mir stehen dort und in $thumbnail dieselben Werte drin. [`get_the_post_thumbnail_url`](https://developer.wordpress.org/reference/functions/get_the_post_thumbnail_url/) ruft unter der Haube doch auch `wp_get_attachment_url` und dann `wp_get_attachment_src` auf.

Reply:
Guter Einwand!